### PR TITLE
config for the service partition

### PR DESCRIPTION
**Pull request by Alek5andr-Kotov**
Config for the service partition

### Changelog entry 

1. The config for the service partition is created without any settings for reading.
2. Auxiliary method `TPartition::IsSupportive`.

### Changelog category 

* Not for changelog (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/core/persqueue/pq_impl.cpp [hunk 1]
```diff
@@ -928,6 +928,22 @@ void TPersQueue::AddSupportivePartition(const TPartitionId& partitionId)
     NewSupportivePartitions.insert(partitionId);
 }
 
+NKikimrPQ::TPQTabletConfig TPersQueue::MakeSupportivePartitionConfig() const
+{
+    NKikimrPQ::TPQTabletConfig partitionConfig = Config;
+
+    partitionConfig.MutableReadRules()->Clear();
+    partitionConfig.MutableReadFromTimestampsMs()->Clear();
+    //partitionConfig.MutableConsumerFormatVersions()->Clear();
```
Comment (alexnick88) [inline ydb/core/persqueue/pq_impl.cpp:937]: uncomment it

File: ydb/core/persqueue/pq_impl.cpp [hunk 2]
```diff
@@ -928,6 +928,22 @@ void TPersQueue::AddSupportivePartition(const TPartitionId& partitionId)
...
+    //partitionConfig.MutableConsumerCodecs()->Clear();
```
Comment (alexnick88) [inline ydb/core/persqueue/pq_impl.cpp:938]: uncompent it

File: ydb/core/persqueue/pq_impl.cpp [hunk 3]
```diff
@@ -936,7 +952,7 @@ void TPersQueue::CreateSupportivePartitionActor(const TPartitionId& partitionId,
     TPartitionInfo& partition = Partitions.at(partitionId);
     partition.Actor = ctx.Register(CreatePartitionActor(partitionId,
                                                         TopicConverter,
-                                                        Config,
+                                                        MakeSupportivePartitionConfig(),
```
Comment (alexnick88) [inline ydb/core/persqueue/pq_impl.cpp:955]: what happens when config is  updated? Nothing?

File: ydb/core/persqueue/pq_impl.cpp (hunk 2 above)
Comment (Alek5andr-Kotov) [reply ydb/core/persqueue/pq_impl.cpp:938]: The fields ConsumerCodecs and ConsumerFormatVersions are not mentioned anywhere else. Are you sure you need them?

File: ydb/core/persqueue/pq_impl.cpp (hunk 3 above)
Comment (Alek5andr-Kotov) [reply ydb/core/persqueue/pq_impl.cpp]: the configuration of the partitions is being updated
https://github.com/ydb-platform/ydb/pull/1807/commits/0717aaeea5854e3e019fc7bf8999d7fab243abbd

Comment (Alek5andr-Kotov) [reply ydb/core/persqueue/pq_impl.cpp:954]: done
https://github.com/ydb-platform/ydb/pull/1807/commits/1deca5dd2c561d6ccb9aefd8a6c2eed128c02b89

File: ydb/core/persqueue/pq_impl.cpp (hunk 1 above)
Comment (Alek5andr-Kotov) [reply ydb/core/persqueue/pq_impl.cpp]: done
https://github.com/ydb-platform/ydb/pull/1807/commits/2c89b1182d6f8e4cbc77cda3c686f3550c23f3f9

File: ydb/core/persqueue/pq_impl.cpp (hunk 2 above)
Comment (Alek5andr-Kotov) [reply ydb/core/persqueue/pq_impl.cpp]: done
https://github.com/ydb-platform/ydb/pull/1807/commits/2c89b1182d6f8e4cbc77cda3c686f3550c23f3f9

**Review by alexnick88 — DISMISSED** [review]

**Review by alexnick88 — APPROVED** [review]

**Pull request merged**